pkg/chains/storage/tekton/tekton.go

```diff
@@ -30,6 +32,7 @@ const (
 	StorageBackendTekton      = "tekton"
 	PayloadAnnotationFormat   = "chains.tekton.dev/payload-%s"
 	SignatureAnnotationFormat = "chains.tekton.dev/signature-%s"
+	CertAnnotationsFormat     = "chains.tekton.dev/cert-%s"
```

Comment:
Do you think we should unify this and do the full "cosign bundle" eventually?

Reply:
that would be cool! I think in cosign the bundle only contains info for [rekor verification](https://github.com/sigstore/cosign/blob/d54af74eceab90005d11cb594665b8a8ef0da431/pkg/cosign/remote/remote.go#L133-L143) and the fulcio cert/chain are separate annotations anyway, so we should still be able to add bundle on top of this.